src/javascript/app_2/App/Components/Elements/ToastMessage/__tests__/close_button.spec.jsx

```diff
@@ -0,0 +1,14 @@
+import React                  from 'react';
+import { expect }             from 'chai';
+import { configure, shallow } from 'enzyme';
+import Adapter                from 'enzyme-adapter-react-16';
+import CloseButton            from '../close_button.jsx';
+
+configure({ adapter: new Adapter() });
+
+describe('CloseButton', () => {
+    it('should render one <CloseButton /> component', () => {
+        const wrapper = shallow(<CloseButton />);
+        expect(wrapper).to.have.length(1);
+    });
+});
```

Comment:
Should we test the required `onClick` propType? I mean, closeButton without close functionality, is just an icon, right? should we check that the component requires onClick prop to be passed? [See here](https://reactjs.org/docs/typechecking-with-proptypes.html)

Reply:
we can add isRequired to the onclick prop but still it passes the test without passing onclick prop, shouldn't it fail?

Reply:
It drops a `console.warn`, we should check out the actual console output or mock it to capture that and assert that it works. Or, we can ignore that part of testing and just make sure that `isRequired` exists in the component's PropTypes.onclick property.